Deleted Team intro page and added a small section in about page

myselfankitaa opened the pull request:
## Description

- Added one link in footer for team
- created page for team short introduction

---


## Screenshots (if applicable)

![Screenshot 2024-12-03 014203](https://github.com/user-attachments/assets/6e873050-f05e-40d1-aa04-f5113891126a)
![Screenshot 2024-12-03 014218](https://github.com/user-attachments/assets/3dee0281-d999-402e-8c3f-be078be3e110)
![Screenshot 2024-12-03 015409](https://github.com/user-attachments/assets/53b5d171-f754-4694-b62a-94f66f7373b7)
![Screenshot 2024-12-03 015429](https://github.com/user-attachments/assets/a09a7f5f-e7ef-45e9-a1da-d421f4d9f24f)

Natata08 commented on frontend/app/team/page.js:
```diff
@@ -0,0 +1,96 @@
+        sx={{ fontSize: { xs: '2rem', sm: '2.5rem', md: '3rem' }, mt: '50px' }}
+        gutterBottom
+      >
+        Team- FONA
```
Maybe it'd better without -, just "Team FONA"

pankaj28843 replied:
`Our Team` is also good.

myselfankitaa commented:
Added a card for team intro in about page and removed specific page for team.
![Screenshot 2024-12-03 104500](https://github.com/user-attachments/assets/6b3927d2-1f24-46ac-be60-4b1979c667a8)
